vignettes/queries_dataset.Rmd

```diff
@@ -0,0 +1,179 @@
+---
+title: "Queries dataset documentation"
```

Comment:
THis should be in title case: "Queries **D**ataset **D**ocumentation"

Reply:
updated